src/match.cpp

```diff
@@ -220,14 +224,12 @@ std::vector<MatchResult> regexbench::match(Engine& engine,
                                   (pLogger ? pLogger.get() : nullptr)));
   }
 
-  uint32_t sec = 0;
-  bool realTime = true;
+  gettimeofday(&begin, NULL);
```

Comment:
Quick idea (unless we do some refactoring) is that just set the same value to the begin time's of all the threads and use them.